src/OrchardCore.Modules/OrchardCore.Roles/Views/Items/SelectUsersInRoleTask.Fields.Edit.cshtml

```diff
@@ -0,0 +1,16 @@
+@using OrchardCore.Roles.Workflows.ViewModels
+@model SelectUsersInRoleTaskViewModel
+
+<div class="mb-3" asp-validation-class-for="OutputKeyName">
+    <label asp-for="OutputKeyName">@T["OutputKeyName"]</label>
+    <input type="text" asp-for="OutputKeyName" class="form-control code" />
+    <span asp-validation-for="OutputKeyName"></span>
+    <span class="hint">@T["The Output Key Name to save list. With Liquid support."]</span>
```

Comment:
Maybe change the test to this 

The designated key name for storing the users' list in the workflow output. This key enables later access to the list within the workflow.With Liquid support.